apps/src/p5lab/spritelab/blocks.js

```diff
@@ -8,29 +8,51 @@ import {APP_HEIGHT, P5LabInterfaceMode} from '../constants';
 import {TOOLBOX_EDIT_MODE} from '../../constants';
 import {animationSourceUrl} from '../animationListModule';
 import {changeInterfaceMode} from '../actions';
-import {Goal, show} from '../AnimationPicker/animationPickerModule';
+import {
+  Goal,
+  show,
+  showBackground
+} from '../AnimationPicker/animationPickerModule';
 import i18n from '@cdo/locale';
 import spritelabMsg from '@cdo/spritelab/locale';
-
-function sprites() {
+function animations(backgrounds) {
```

Comment:
it isn't clear what this param does by its name -- maybe `skipBackgrounds` again? or `includeBackgrounds` if we want the inverse (also open to suggestions)

Reply:
how about isBackgroundAnimations?

Reply:
ah, i still misunderstood the param meaning lol. i think that's fine, or even just `areBackgrounds` since the function is called `animations` and now the param will communicate that it should be a boolean